api/test/robot_tests/web_on_chain/web_on_chain.robot

```diff
@@ -46,14 +46,33 @@ POST /websiteUploader/upload
 
     Should Be Equal As Strings    ${response.json()['address']}    ${WEBSITE_ADDRESS}
 
-GET /all/domains 
-    ${response}=    GET
-    ...    ${API_URL}/all/domains
-    ...    expected_status=${STATUS_OK}
-    Should Contain    ${response.text}    flappy
-
 GET /my/domains/{nickname}
     ${response}=    GET
     ...    ${API_URL}/my/domains/${WALLET_NICKNAME}
     ...    expected_status=${STATUS_OK}
     Should Contain    ${response.text}    ${website_url}
+
+Fetch Current DNS
+    ${response}=    GET    
+    ...    ${API_URL}/massa/node
+    ...    expected_status=${STATUS_OK}
+    ${current_dns}    Set Variable    ${response.json()['dns']}
+    Set Global Variable    ${DNS}    ${current_dns}
+        
+POST /cmd/executeFunction To remove the dns entry
+    ${argument}=    keywords.String To Arg    ${website_url}
+    ${data}=    Create Dictionary
```

Comment:
`Create Dictionary` what is this used for ?

Reply:
we create the Dictionary which contains all inputs needed for the callSC   

    ...    name=deleteEntryFromDNS
    ...    at=${DNS}
    ...    args=${argument}`

Reply:
this is how we create an associative array